man/ostree-admin-deploy.xml

```diff
@@ -121,6 +121,16 @@ License along with this library. If not, see <https://www.gnu.org/licenses/>.
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--lock-finalization</option></term>
+
+                <listitem><para>
+                    The deployment will not be "finalized" by default on shutdown; to later
+                    queue it, use <literal>ostree admin unlock-finalization</literal>.
```

Comment:
```suggestion
                    queue it, use <literal>ostree admin lock-finalization --unlock</literal>.
```

Reply:
Good catch, thanks for the careful review!